src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`styles/main.scss`). SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure that you link to the compiled CSS file instead.